startup_script/startup.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/sh
 
-echo "Running sample startup script."
\ No newline at end of file
+echo 'Running sample startup script.'
+
+chmod 400 ${AIRFLOW_LOCAL_CONNECTIONS_DIR:-${PWD}}/files/keys/ssh_host.pem;
```

Comment:
looks like we are putting keys in keys directory inside files directory.
Can we do for snowflake connection as well(private_key) ->  example_connections/example_snowflake_connections.json
Also we need to update [doc](https://seesaw-eng.atlassian.net/wiki/spaces/DP/pages/88473854/Airflow+Development+Guide#%5BinlineExtension%5D%5BhardBreak%5DPrerequisites) where we mentioned about it if we make changes.

Reply:
Left you a comment in confluence, but feel free to update there

Reply:
@morrie-seesaw did you cleanup testing rds user and schema which you shared in slack ? I'm unable to connect to rds with that user.

Reply:
Is this line `chmod 400` worked for you ?
I'm getting error:
![image](https://github.com/user-attachments/assets/0610c803-a8dd-4c65-a200-15ddcdd32b1e)